libparsec/crates/testbed/src/template/build.rs

```diff
@@ -228,9 +230,17 @@ impl TestbedTemplateBuilder {
             events: template.events.clone(),
             stuff: template.stuff.clone(),
             counters: template.build_counters.clone(),
+            check_consistency: true,
         }
     }
 
+    pub fn with_check_consistency_disabled<T>(&mut self, cb: impl FnOnce(&mut Self) -> T) -> T {
```

Comment:
```suggestion
    pub fn without_consistency<T>(&mut self, cb: impl FnOnce(&mut Self) -> T) -> T {
```

:shrug:

Reply:
@touilleMan chose the name, he didn't want the `without`